frontend/src/lib/components/CommandPalette/commandPaletteLogic.ts

```diff
@@ -578,7 +570,7 @@ export const commandPaletteLogic = kea<
                             },
                         })
                     )
-                    if (isURL(argument))
+                    if (argument && isURL(argument))
```

Comment:
Any reason for removing this? My guess is that this was added due to TS linting.

Reply:
TS linter was complaining that `argument` could be `undefined` and trying to call `isURL` on `undefined` would fail.